backend/access/cbac.py

```diff
@@ -135,7 +136,7 @@ def is_graphql_allowed_for_model(
 def graphql_check_access(instance, info, field: str):
     user = info.context.user
     operation = "query"
-    extra = dict(slug=instance.slug) if hasattr(instance, "slug") else {}
+    extra = {"slug": instance.slug} if hasattr(instance, "slug") else {}
```

Comment:
This is a personal preference and I will keep writing `dict(foo=5)` :) But no need to remove this from the PR.

Reply:
Yeah, this got changed automagically somewhere else as well, and I reverted it then too. Will revert this too while rebasing, no biggie!

Reply:
Also, the personal preference is noted by
```toml
ignore = [
  # ...
  "C408",  # unnecessary dict call (these are likely stylistic)
]
```
😌